robots/flakefinder/index_test.go

```diff
@@ -66,12 +66,16 @@ var _ = Describe("index.go", func() {
 
 	When("writing the index page", func() {
 
+		printLogOutput := false
```

Comment:
This was left here for debug purpouses I guess. Should you remove it?

Reply:
Left it here to be able to control the output manually. Although it's possible to unit test the output the effort is not in relation I think.

Reply:
Looks like a good candidate for the commandline. :+1:

Reply:
> By doing this you'll need to change the code manually to see the output.
> A better way is to make it a global, driven by a flag (eventually read / parsed by tests if you plan to use it only in tests)

Right! Created #206 to address this.